scripts/docker/debian.8/Dockerfile

```diff
@@ -7,60 +7,40 @@
 FROM debian:jessie
 
 # Misc Dependencies for build
-RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+RUN apt-get update && \
+    apt-get -qqy install \
+        curl \
+        unzip \
+        gettext \
+        sudo && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
This will likely help reduce the size of our docker images on disk, thanks for doing this.